### ISSUE TITLE
Creation of new entity role 

### ISSUE THREAD

**Issue by cdantec** [bio-tools/biotoolsRegistry#495]
Hi!
in Credits&Support section, is it possible to have a new entity role : 'Annotator'  ?

**Comment by hansioan** [comment]
Hi,

What would be the purpose of the 'Annotator' role? The definitions of the existing roles can be found at:
https://biotools.readthedocs.io/en/latest/curators_guide.html#entity-role

Perhaps one of the existing ones can satisfy the role you need?
For example the 'Documentor' role has the definition `Author of software documentation including making edits to a bio.tools entry.`  Would that be ok?

**Comment by joncison** [comment]
This sound like the same thing to me, Annotator == Documentor ?

**Comment by cdantec** [comment]
Nop!
 for us annotator is the person who annote data.
annotator or curator if  you prefer. For ANISEED there is a person who read all the biblioagraphy, extract the information to annotate data in our database.

**Comment by jvanheld** [comment]
Indeed, I agree that we should make a strong distinction between the general use of annotator in the field of bioinformatics (someone who populates a biological database with biological information) and the role of "tool annotator" (someone who annotates a tool in bio.tool).

**Comment by joncison** [comment]
I understand now.  These are of course very different things.  I'd suggest something like:

* _Curator_ -  The author of annotations on entries within a data resource. 
* _Documentor_ - Author of software documentation including making edits to a bio.tools entry.

or something like that?

Once we agree terms & definitions we can ping an issue to https://github.com/bio-tools/biotoolsSchema/issues to implement the change in the schema.  @hansioan - the above implies a simple addition to the enum (== non-breaking change) 

PS.  very good & important to do even small things to raise profile of curators!

**Comment by cdantec** [comment]
I am not comfortable with this definition because the expertise of a (bio) curator is not sufficiently valued.
Someone who would do automatic annotation on a large scale would be included in this definition which seems to me 2 different areas of expertise.
I found this on the website on the International Society of Biocuration (https://www.biocuration.org)
Biocurator "involves the translation and integration of information relevant to biology into a database or resource that enables integration of the scientific literature "

Could that make sense to you?

**Comment by jvanheld** [comment]
Great. I think "**biocurator**" should definitely be included as a term, and as you say it is important to valorise the work of biocurators, which is a profession per se (among many other database, the value of Swissprot comes from the hundreds of biocurators who spend 100% of their time to curate biological data since several decades). 

And to avoid any confusion we might specify "**tool curator**" for the people who curate tools in bio.tools.

**Comment by matuskalas** [comment]
I think I understand @cdantec, and I can imagine a scenario where the distinction would be useful. To decrease the danger of misunderstanding, I'd suggest rephrasing the naming and definitions to something like:

 * _Documentor_ - Author of software documentation including tools annotation in _bio.tools_ or other tool information resources
 * _Data curator_ - Persom organising and integrating data and information from various sources, including annotation, publication, presentation, and availability of the data for reuse and preservation.

Is this making sense, @cdantec?

Note: Definition "gisted" & derived from https://en.wikipedia.org/wiki/Data_curation. Data curator is an important contributor to tools such as data resources, or tools relying heavily on underlying data (including machine learning training sets).

On the other hand, I would leave it open for further discussion and research within the broad community, whether we need so specific granularity inside _bio.tools_ records. Would it mean we then also need specific contributor roles for PIs, MDs taking samples from patients, lab technicians, sequencing facility operators, data analysts, _etc._?

Or should these all be just unspecific _Contributors_? I'd also advocate, that whatever additional granularity we're going to maintain or add, we should then reuse some contributor roles standards that are emerging in the modern (bio)scientific publishing domain.

**Comment by joncison** [comment]
For now, I think we should add the term, _Biocurator_ or (the more general) _Data curator_, and see what else comes in due course. But not leave them as unspecified _Contributors_ (the DB folks deserve some love :).

**Comment by cdantec** [comment]
make sense, yes.
I totally agree that too much granularity would not add value..

I focus on the specific job of biocurators (certainly because they bring real added value to the quality of the for the research community) but also in this specific case because it is an integral part of elixir: (Data platform, Task 3. Scalable Curation).
They allow to use and re-use the generated data, they develop standards. They have a very strong part in the philosophy of Elixir,  that's why it seems important to me :-)

**Comment by matuskalas** [comment]
I'm sure the other roles would just as well emphasise their importance. 😉

Anyhow, could we then please at least make sure we call them with the more precise and broadly understadable term _Data curators_, instead of _Biocurators_?

**Comment by jvanheld** [comment]
I think that the term biocurator is quite well-defined. It is a subset of the class "data curator", whose expertise is in the field of biology. Since in bio.tools we also call curators the people who are curating the tools entries of bio.tools, I think the term biocurator would help to alleviate the possible confusion between two possible intervention on the same record that would describes a biological database:
- the person who entered the biological data in the database (biocurator)
- the person who entered the data about the database in bio.tools (tool curator, or, even more explicitly, bio.tools curator)


Cheers

Jacques

**Comment by matuskalas** [comment]
> * the person who entered the biological data in the database (biocurator)
> 
> * the person who entered the data about the database in bio.tools (tool curator, or, even more explicitly, bio.tools curator)

Exactly, that's why I prefer _Data curator_, as opposed to _Biocurator_, _Bio(data)curator_, _Bio(tools)curator_, _Bio(whatever)curator_. In addition, _Data curator_ is reusable for other data curators than only curators of data about biological macromolecules (_e.g._ also the "data stewards").
